convert2rhel/actions/system_checks/is_loaded_kernel_latest.py

```diff
@@ -96,6 +110,14 @@ def run(self):  # pylint: disable= too-many-return-statements
                 "Couldn't fetch the list of the most recent kernels available in "
                 "the repositories. Skipping the loaded kernel check."
             )
+            self.add_message(
+                level="WARNING",
+                id="UNABLE_TO_FETCH_RECENT_KERNELS",
+                message=(
+                    "Couldn't fetch the list of the most recent kernels available in "
+                    "the repositories. Skipping the loaded kernel check."
+                ),
+            )
```

Comment:
Remember to remove the logger.warning above

```suggestion
            message = (
                    "Couldn't fetch the list of the most recent kernels available in "
                    "the repositories. Skipping the loaded kernel check."
                )
            logger.warning(message)
            self.add_message(
                level="WARNING",
                id="UNABLE_TO_FETCH_RECENT_KERNELS",
                message=message,
            )
```